Microsoft.Toolkit.Uwp.SampleApp/SamplePages/ThemeListener/ThemeListenerPage.xaml.cs

```diff
@@ -16,9 +17,8 @@ public sealed partial class ThemeListenerPage : Page
         public ThemeListenerPage()
         {
             this.InitializeComponent();
-            Listener = new ThemeListener();
             this.Loaded += ThemeListenerPage_Loaded;
-            Listener.ThemeChanged += Listener_ThemeChanged;
+            SampleController.Current.ThemeListener.ThemeChanged += Listener_ThemeChanged;
             SampleController.Current.ThemeChanged += Current_ThemeChanged;
```

Comment:
We should just need the one event here. Not sure why we expose the `ThemeListener` publicly on the `SampleController` as we expose everything via methods. For instance the `SampleController.Current.ThemeChanged` is invoked the same as the ThemeListener:

https://github.com/CommunityToolkit/WindowsCommunityToolkit/blob/bdd1d84dc079de3d640d95a2a963c3e37cbc3ccc/Microsoft.Toolkit.Uwp.SampleApp/Pages/SampleController.xaml.cs#L41-L43

If we make the `ThemeListener` on `SampleController` private what else breaks (once we fix this sample)?

Reply:
I added the public `ThemeListener` property to the Sample Controller (see above).  

Since we're just interested in returning the data of a property, and not performing any computation, a property seemed more natural, as well as heeding advice from the  [Choosing Between Properties and Methods](https://docs.microsoft.com/en-us/previous-versions/dotnet/netframework-4.0/ms229054(v=vs.100)?redirectedfrom=MSDN) section of Design Guidelines for Developing Class Libraries (**note**: online doc is in VB, not C#, so shield your eyes accordingly):  

> In general, methods represent actions and properties represent data. Properties are meant to be used like fields, meaning that properties should not be computationally complex or produce side effects. When it does not violate the following guidelines, consider using a property, rather than a method, because less experienced developers find properties easier to use.

Reply:
For consistency in the Sample Controller, I can convert the public property to a method, instead